src/main/java/com/example/bookstore/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,20 @@
+package com.example.bookstore.dto.user;
+
+import com.example.bookstore.validation.FieldsMatch;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+@FieldsMatch(
+        field = "password",
+        fieldMatch = "repeatedPassword"
+)
+public record UserRegistrationRequestDto(
+        @NotBlank @Email String email,
+        @NotBlank @Size(min = 3) String firstName,
+        @NotBlank @Size(min = 3) String lastName,
```

Comment:
you can leave it if you want but why do you need min size?

Reply:
I thought that it would be strange if we had user with a name. length < 3